### PR TITLE
Custom formatter documentation update

### PR DESCRIPTION
**Pull request by shagra-ms**
## Category

- [x] Content fix
- [ ] New article

## Related issues

- Custom formatter documentation update

## What's in this Pull Request?

The PR captures the following updates -
1. New `openContextMenu` action
2. Managed Metadata column support
3. View formatting enhancement - group header, footer and list footer formatter support

### CODE REVIEW

**Comment by VesaJuvonen** [comment]
Docs Build status updates of commit _[9e80d53](https://github.com/shagra-ms/sp-dev-docs/commits/9e80d539843f2f147adbb942efec387d3a7ab660)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/declarative-customization/view-formatting.md](https://github.com/shagra-ms/sp-dev-docs/blob/shre/cf-docs-update-032021/docs/declarative-customization/view-formatting.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/declarative-customization/view-formatting?branch=pr-en-us-6832) | [Details](#user-content-fb5713e9dcc2d94ce97edac0dc112a95-0a3995f2)
[docs/declarative-customization/column-formatting.md](https://github.com/shagra-ms/sp-dev-docs/blob/shre/cf-docs-update-032021/docs/declarative-customization/column-formatting.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/declarative-customization/column-formatting?branch=pr-en-us-6832) |
[docs/images/employee-formatted-group-header-aggregate-gallery-layout.png](https://github.com/shagra-ms/sp-dev-docs/blob/shre/cf-docs-update-032021/docs/images/employee-formatted-group-header-aggregate-gallery-layout.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/declarative-customization/view-formatting?branch=pr-en-us-6832) |
[docs/images/employee-formatted-group-header-aggregate-list-layout.png](https://github.com/shagra-ms/sp-dev-docs/blob/shre/cf-docs-update-032021/docs/images/employee-formatted-group-header-aggregate-list-layout.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/declarative-customization/view-formatting?branch=pr-en-us-6832) |
[docs/images/employee-formatted-group-header-gallery-layout.png](https://github.com/shagra-ms/sp-dev-docs/blob/shre/cf-docs-update-032021/docs/images/employee-formatted-group-header-gallery-layout.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/declarative-customization/view-formatting?branch=pr-en-us-6832) |
[docs/images/employee-formatted-group-header-list-layout.png](https://github.com/shagra-ms/sp-dev-docs/blob/shre/cf-docs-update-032021/docs/images/employee-formatted-group-header-list-layout.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/declarative-customization/view-formatting?branch=pr-en-us-6832) |
[docs/images/group-footer-list-layout.png](https://github.com/shagra-ms/sp-dev-docs/blob/shre/cf-docs-update-032021/docs/images/group-footer-list-layout.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/declarative-customization/view-formatting?branch=pr-en-us-6832) |
[docs/images/list-footer-list-layout.png](https://github.com/shagra-ms/sp-dev-docs/blob/shre/cf-docs-update-032021/docs/images/list-footer-list-layout.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/declarative-customization/view-formatting?branch=pr-en-us-6832) |

<a id="fb5713e9dcc2d94ce97edac0dc112a95-0a3995f2"></a>
### [docs/declarative-customization/view-formatting.md](https://github.com/shagra-ms/sp-dev-docs/blob/shre/cf-docs-update-032021/docs/declarative-customization/view-formatting.md)
 - **Line 861, Column 101**: **[Warning-[hard-coded-locale](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://docs.microsoft.com/en-us/sharepoint/dev/declarative-customization/column-formatting#special-string-values' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 898, Column 106**: **[Warning-[hard-coded-locale](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://docs.microsoft.com/en-us/sharepoint/dev/declarative-customization/column-formatting#foreach' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C3%5C18%5C8c4da01d-e430-7fe7-0bea-2c4100cce9e8%5CPullRequest%5C202103181319378613-6832%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=81U59fGDcHAzA3aRjODIIK2fz2Q8KyRFbLnc5B%2FnYY8%3D&st=2021-03-18T13%3A16%3A04Z&se=2021-04-18T13%3A21%3A04Z&sp=r).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=master) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by VesaJuvonen** [comment]
Docs Build status updates of commit _[1ffbb50](https://github.com/shagra-ms/sp-dev-docs/commits/1ffbb5036016c74f59b1c1cb3f18ef331756c972)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/declarative-customization/column-formatting.md](https://github.com/shagra-ms/sp-dev-docs/blob/shre/cf-docs-update-032021/docs/declarative-customization/column-formatting.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/declarative-customization/column-formatting?branch=pr-en-us-6832) |
[docs/declarative-customization/view-formatting.md](https://github.com/shagra-ms/sp-dev-docs/blob/shre/cf-docs-update-032021/docs/declarative-customization/view-formatting.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/declarative-customization/view-formatting?branch=pr-en-us-6832) |
[docs/images/employee-formatted-group-header-aggregate-gallery-layout.png](https://github.com/shagra-ms/sp-dev-docs/blob/shre/cf-docs-update-032021/docs/images/employee-formatted-group-header-aggregate-gallery-layout.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/declarative-customization/view-formatting?branch=pr-en-us-6832) |
[docs/images/employee-formatted-group-header-aggregate-list-layout.png](https://github.com/shagra-ms/sp-dev-docs/blob/shre/cf-docs-update-032021/docs/images/employee-formatted-group-header-aggregate-list-layout.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/declarative-customization/view-formatting?branch=pr-en-us-6832) |
[docs/images/employee-formatted-group-header-gallery-layout.png](https://github.com/shagra-ms/sp-dev-docs/blob/shre/cf-docs-update-032021/docs/images/employee-formatted-group-header-gallery-layout.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/declarative-customization/view-formatting?branch=pr-en-us-6832) |
[docs/images/employee-formatted-group-header-list-layout.png](https://github.com/shagra-ms/sp-dev-docs/blob/shre/cf-docs-update-032021/docs/images/employee-formatted-group-header-list-layout.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/declarative-customization/view-formatting?branch=pr-en-us-6832) |
[docs/images/group-footer-list-layout.png](https://github.com/shagra-ms/sp-dev-docs/blob/shre/cf-docs-update-032021/docs/images/group-footer-list-layout.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/declarative-customization/view-formatting?branch=pr-en-us-6832) |
[docs/images/list-footer-list-layout.png](https://github.com/shagra-ms/sp-dev-docs/blob/shre/cf-docs-update-032021/docs/images/list-footer-list-layout.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/declarative-customization/view-formatting?branch=pr-en-us-6832) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C3%5C18%5C8c4da01d-e430-7fe7-0bea-2c4100cce9e8%5CPullRequest%5C202103181338357491-6832%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=CzfXAf%2FE9JjMwENXwp5iIhRunS0UNnfKgD1jFY5D%2Bwg%3D&st=2021-03-18T13%3A34%3A43Z&se=2021-04-18T13%3A39%3A43Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by VesaJuvonen** [comment]
Docs Build status updates of commit _[ad090eb](https://github.com/shagra-ms/sp-dev-docs/commits/ad090eb1466e4c16a10fac15223548067bdbb888)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/declarative-customization/column-formatting.md](https://github.com/shagra-ms/sp-dev-docs/blob/shre/cf-docs-update-032021/docs/declarative-customization/column-formatting.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/declarative-customization/column-formatting?branch=pr-en-us-6832) |
[docs/declarative-customization/view-formatting.md](https://github.com/shagra-ms/sp-dev-docs/blob/shre/cf-docs-update-032021/docs/declarative-customization/view-formatting.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/declarative-customization/view-formatting?branch=pr-en-us-6832) |
[docs/images/employee-formatted-group-header-aggregate-gallery-layout.png](https://github.com/shagra-ms/sp-dev-docs/blob/shre/cf-docs-update-032021/docs/images/employee-formatted-group-header-aggregate-gallery-layout.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/declarative-customization/view-formatting?branch=pr-en-us-6832) |
[docs/images/employee-formatted-group-header-aggregate-list-layout.png](https://github.com/shagra-ms/sp-dev-docs/blob/shre/cf-docs-update-032021/docs/images/employee-formatted-group-header-aggregate-list-layout.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/declarative-customization/view-formatting?branch=pr-en-us-6832) |
[docs/images/employee-formatted-group-header-gallery-layout.png](https://github.com/shagra-ms/sp-dev-docs/blob/shre/cf-docs-update-032021/docs/images/employee-formatted-group-header-gallery-layout.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/declarative-customization/view-formatting?branch=pr-en-us-6832) |
[docs/images/employee-formatted-group-header-list-layout.png](https://github.com/shagra-ms/sp-dev-docs/blob/shre/cf-docs-update-032021/docs/images/employee-formatted-group-header-list-layout.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/declarative-customization/view-formatting?branch=pr-en-us-6832) |
[docs/images/group-footer-list-layout.png](https://github.com/shagra-ms/sp-dev-docs/blob/shre/cf-docs-update-032021/docs/images/group-footer-list-layout.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/declarative-customization/view-formatting?branch=pr-en-us-6832) |
[docs/images/list-footer-list-layout.png](https://github.com/shagra-ms/sp-dev-docs/blob/shre/cf-docs-update-032021/docs/images/list-footer-list-layout.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/declarative-customization/view-formatting?branch=pr-en-us-6832) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C3%5C18%5C8c4da01d-e430-7fe7-0bea-2c4100cce9e8%5CPullRequest%5C202103181406414900-6832%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=uo51ZSZZTOQugXGLOd2Jys5px%2FzkAx4i15MH53bzssM%3D&st=2021-03-18T14%3A02%3A49Z&se=2021-04-18T14%3A07%3A49Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/declarative-customization/view-formatting.md [hunk 1]
```diff
@@ -154,13 +155,306 @@ This example uses the `rowFormatter` element, which totally overrides the render
...
 
+### Build custom group headers and footers
+
+You can use `groupProps` to format group headers with flexibility to add grouped column's data, display name and item count. You can also add group aggregates in the group headers or format it directly in the group footers.
+
+#### Example: Color coded group header
+
+In the example below we have list with group headers formatted according to column metadata.
+![Employee list grouped by City with formatted group header](../images/employee-formatted-group-header-list-layout.png)
+
```
Comment (niketjain2020) [inline docs/declarative-customization/view-formatting.md:169]: We should write some commentary about the JSON code here to make user aware - just like above other JSON snippets like use of groupProps and header formatter

File: docs/declarative-customization/view-formatting.md [hunk 2]
```diff
@@ -154,13 +155,306 @@ This example uses the `rowFormatter` element, which totally overrides the render
...
+            }
+          ]
+        }
+      ]
+    }
+  }
+}
+```
+
+#### Example: Color coded group header with aggregates
```
Comment (niketjain2020) [inline docs/declarative-customization/view-formatting.md:255]: Either - Color coded group headers with aggregates or Color coded group header with aggregate

File: docs/declarative-customization/view-formatting.md [hunk 3]
```diff
@@ -154,13 +155,306 @@ This example uses the `rowFormatter` element, which totally overrides the render
...
+
+In the example below we have list with group headers formatted with group aggregates.
```
Comment (niketjain2020) [inline docs/declarative-customization/view-formatting.md:257]: single line space between image and description

File: docs/declarative-customization/view-formatting.md [hunk 4]
```diff
@@ -154,13 +155,306 @@ This example uses the `rowFormatter` element, which totally overrides the render
...
+![Employee list grouped by City with formatted group header with Aggregates summary](../images/employee-formatted-group-header-aggregate-list-layout.png)
+
+In this example the `hideFooter` for `groupProps` is set to `true` to hide the group footer and the `@aggregates` array is used to display a summary in the group header.
```
Comment (niketjain2020) [inline docs/declarative-customization/view-formatting.md:260]: In this example the `hideFooter` for `groupProps` are set to `true` - to hide the group footer and the `@aggregates` array is used to display a summary in the group header.

File: docs/declarative-customization/view-formatting.md [hunk 5]
```diff
@@ -154,13 +155,306 @@ This example uses the `rowFormatter` element, which totally overrides the render
...
+      ]
+    }
+  }
+}
+
+```
+
+#### Example: Custom group footer
+
+In the example below we have list with group footer formatted according to aggregate value.
```
Comment (niketjain2020) [inline docs/declarative-customization/view-formatting.md:352]: Leave a line equivalent space

File: docs/declarative-customization/view-formatting.md [hunk 6]
```diff
@@ -154,13 +155,306 @@ This example uses the `rowFormatter` element, which totally overrides the render
...
+![Employee list grouped by City with formatted group footer](../images/group-footer-list-layout.png)
+
+In this example the `footerFormatter` for `groupProps` is set to format the group footer and the `@columnAggregate` is used to access column aggregate.
```
Comment (niketjain2020) [inline docs/declarative-customization/view-formatting.md:355]: In this example the `footerFormatter` for `groupProps` are used to format the group footer and the `@columnAggregate` is used to access column aggregate.

File: docs/declarative-customization/view-formatting.md [hunk 7]
```diff
@@ -154,13 +155,306 @@ This example uses the `rowFormatter` element, which totally overrides the render
...
+}
+```
+
+### Build custom list footers
+
+You can use `footerFormatter` to format list footer with access to column aggregates.
+
+#### Example: Custom list footer
+
+In the example below we have list with footer formatted according to aggregate value.
```
Comment (niketjain2020) [inline docs/declarative-customization/view-formatting.md:398]: 1) Line equivalent space 
2) In the example below we have list with formatted footer as per the aggregate value.

File: docs/declarative-customization/view-formatting.md [hunk 8]
```diff
@@ -262,52 +556,363 @@ The following image shows a customized card in Gallery layout:
 }
 ```
 
+### Build custom group headers
+
+You can use `groupProps` to format group headers with flexibility to add grouped column's data, display name and item count. You can also add group aggregates in the group headers.
+
+#### Example: Color coded group header
+
+In the example below we have gallery with group headers formatted according to column metadata.
```
Comment (niketjain2020) [inline docs/declarative-customization/view-formatting.md:565]: In the example below we have gallery view with formatted group headers as per column metadata.

Line equivalent space

File: docs/declarative-customization/view-formatting.md [hunk 9]
```diff
@@ -262,52 +556,363 @@ The following image shows a customized card in Gallery layout:
...
+### Build custom group headers
+
+You can use `groupProps` to format group headers with flexibility to add grouped column's data, display name and item count. You can also add group aggregates in the group headers.
+
+#### Example: Color coded group header
+
+In the example below we have gallery with group headers formatted according to column metadata.
+![Employee gallery grouped by City with formatted group header](../images/employee-formatted-group-header-gallery-layout.png)
+
+Note that gallery card formatter is skipped in the below JSON for simplicity.
```
Comment (niketjain2020) [inline docs/declarative-customization/view-formatting.md:568]: There is a particular format to put a box around and change bgc for notes to highlight, can we do that ?

File: docs/declarative-customization/view-formatting.md [hunk 10]
```diff
@@ -262,52 +556,363 @@ The following image shows a customized card in Gallery layout:
...
+        }
+      ]
+    }
+  }
+}
+```
+
+#### Example: Color coded group header with aggregates
+
+In the example below we have list with group headers formatted with group aggregates.
```
Comment (niketjain2020) [inline docs/declarative-customization/view-formatting.md:671]: In the example below we have list with formatted group headers formatted and group aggregates.

line equivalent space

**Review by arkogupta — APPROVED** [review]
groupProps part looks good to me!

**Comment by VesaJuvonen** [comment]
Docs Build status updates of commit _[0beeab7](https://github.com/shagra-ms/sp-dev-docs/commits/0beeab76863a49c619526e124df6a4e6f9ce75f4)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/declarative-customization/column-formatting.md](https://github.com/shagra-ms/sp-dev-docs/blob/shre/cf-docs-update-032021/docs/declarative-customization/column-formatting.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/declarative-customization/column-formatting?branch=pr-en-us-6832) |
[docs/declarative-customization/view-formatting.md](https://github.com/shagra-ms/sp-dev-docs/blob/shre/cf-docs-update-032021/docs/declarative-customization/view-formatting.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/declarative-customization/view-formatting?branch=pr-en-us-6832) |
[docs/images/employee-formatted-group-header-aggregate-gallery-layout.png](https://github.com/shagra-ms/sp-dev-docs/blob/shre/cf-docs-update-032021/docs/images/employee-formatted-group-header-aggregate-gallery-layout.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/declarative-customization/view-formatting?branch=pr-en-us-6832) |
[docs/images/employee-formatted-group-header-aggregate-list-layout.png](https://github.com/shagra-ms/sp-dev-docs/blob/shre/cf-docs-update-032021/docs/images/employee-formatted-group-header-aggregate-list-layout.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/declarative-customization/view-formatting?branch=pr-en-us-6832) |
[docs/images/employee-formatted-group-header-gallery-layout.png](https://github.com/shagra-ms/sp-dev-docs/blob/shre/cf-docs-update-032021/docs/images/employee-formatted-group-header-gallery-layout.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/declarative-customization/view-formatting?branch=pr-en-us-6832) |
[docs/images/employee-formatted-group-header-list-layout.png](https://github.com/shagra-ms/sp-dev-docs/blob/shre/cf-docs-update-032021/docs/images/employee-formatted-group-header-list-layout.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/declarative-customization/view-formatting?branch=pr-en-us-6832) |
[docs/images/group-footer-list-layout.png](https://github.com/shagra-ms/sp-dev-docs/blob/shre/cf-docs-update-032021/docs/images/group-footer-list-layout.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/declarative-customization/view-formatting?branch=pr-en-us-6832) |
[docs/images/list-footer-list-layout.png](https://github.com/shagra-ms/sp-dev-docs/blob/shre/cf-docs-update-032021/docs/images/list-footer-list-layout.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/declarative-customization/view-formatting?branch=pr-en-us-6832) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C3%5C22%5C8c4da01d-e430-7fe7-0bea-2c4100cce9e8%5CPullRequest%5C202103220954159996-6832%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=zV4hcY%2FBhBBtqtKWizSnKuAfyuN1RQ%2B8fk2JJNMaCOQ%3D&st=2021-03-22T09%3A50%3A26Z&se=2021-04-22T09%3A55%3A26Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Pull request merged**